docs/getting-started.md

```diff
@@ -3,6 +3,48 @@ sidebar_position: 1
 slug: /
 ---
 
-# Getting Started
+## How to Build a Guided Project
 
-Welcome to the **Author Workbench Knowledgebase**. We are in the process of building this knowledgebase to help you build **amazing courses and guided projects** with Author Workbench.
+Congratulations on creating your Guided Project. This article will walk you through the steps to create and publish a Guided Project.
+
+## Prerequisites
+
+If you haven't went through our onboarding process, please visit https://www.skills.network/authoring/ first and follow the instructions.
+
+## Get Started
+
+Upon creating a project, you should see the following screen:
+<img width="1904" alt="guided project creation" src="https://user-images.githubusercontent.com/46271636/171186797-e3a65a85-99a4-4bc6-9be0-ee30a5004e46.png">
+
+You will see three major components:
+
+- GitLab Project
+- Asset Library
+- Skills Network Studio
+
+## Authoring
+
+### GitLab Project
+
+This is where most of the work is done and where you will author the content of your Guided Project. The GitLab project is created with a template, so make sure you go through the `README.md` file in your GitLab project.
+
+### Asset Library
+
+Whenever you make any changes to your content on the GitLab project, they will be published to asset library and a public URL will be generated. You should only share assets with links you obtain from asset library with your learners.
+
+Clicking `Open Asset Library` will bring you to the following page. Hovering your mouse over the `instructions.md` file allows you to test your content in the Skills Network Lab environment. Clicking the `clipboard` icon will copy the public URL of your content to your clipboard. You will be needing this link when publishing your Guided Project in the section below.
+![asset library](https://user-images.githubusercontent.com/46271636/171186922-3de7649b-6cb9-474a-9dd4-8f0808182f39.png)
+
+## Publishing
+
+### Submit for Review
+
+Once you are done authoring your content and are ready to make it publicly available through our platform. Go back to your project on Author Workbench and click `Publish`.
+![publish](https://user-images.githubusercontent.com/46271636/171187019-b45f6250-d18e-4991-8fe0-96e735c903e0.png)
+
+You are required to include a release note for every new version. Note that this text will be made available to all learners, so write something that makes sense to your learners.
```

Comment:
This whole section will have to be completely re-written. The 3 major components are now "Edit Instructions", "Test your Project" and "Get Published". Will be easier once the new UI is up again

Wilson will also be working on a separate guide for how to publish, so instead of writing that here we can just link to his guide when it's ready